src/game/gamecore.h

```diff
@@ -18,6 +18,7 @@ class CTuneParam
 public:
 	void Set(int v) { m_Value = v; }
 	int Get() const { return m_Value; }
+	float GetLua() const { return m_Value/100.0f; }
```

Comment:
The function name prefix `…Lua` is meant to be used for functions bound into the API where the original function can not be used directly, whereas this here is just a helper function that divides the value by 100 which could also be done at the only spot in the code where this function is used.
Please remove it and use `((float)m_##Name.Get())/100.0f` in line 42 instead.